src/sql/src/plan/func.rs

```diff
@@ -520,6 +529,9 @@ pub enum ParamType {
     /// except it does not permit either `ScalarType::Array` or
     /// `ScalarType::List`.
     NonVecAny,
+    /// A polymorphic pseudotype permitting a `ScalarType::Map` of any element
```

Comment:
any value type?

Reply:
I've updated it to any non-nested value type!